### PR TITLE
205 reject message

### PR DESCRIPTION
**Pull request by palagdan**

### CODE REVIEW

File: src/components/record/RecordRow.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { RECORD_PHASE, ROLE } from "../../constants/DefaultConstants";
 import { useI18n } from "../../hooks/useI18n";
 import { IfGranted } from "react-authorization";
 import PromiseTrackingMask from "../misc/PromiseTrackingMask";
+import rejectButton from "../button/RejectButton.jsx";
```
Comment (blcham) [inline src/components/record/RecordRow.jsx:10]:
```suggestion
import rejectButton from "../button/RejectButton.jsx";
```

Comment (blcham) [reply src/components/record/RecordRow.jsx:10]: I do not see this used.

**Review by blcham — CHANGES_REQUESTED** [review]
LGTM, just minor issue

File: src/components/record/RecordRow.jsx (hunk 1 above)
Comment (palagdan) [reply src/components/record/RecordRow.jsx]: Thanks, it was removed

**Pull request merged**